src/main/java/org/eclipse/cargotracker/interfaces/booking/facade/BookingServiceFacade.java

```diff
@@ -28,4 +30,8 @@
 	List<Location> listShippingLocations();
 
 	List<CargoRoute> listAllCargos();
+
+	List<TrackingId> listAllTrackingIds();
```

Comment:
This should probably just a list of strings. In general, domain objects should not be exposed by the facade.